lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_EnsembleReordering.py

```diff
@@ -604,7 +604,7 @@ def test_basic(self):
     def test_works_for_different_percentile_coord(self):
         """Test that it still works for different percentile coordinate"""
         cube = self.post_processed_percentiles
-        cube.coord(self.perc_coord).rename("percentile_over_dummy")
```

Comment:
Could probably get rid of this test now that there is only one "percentile" coordinate name.

Reply:
Done